File: appointments-web/AppointmentApp.Core/Advisers/Adviser.cs
using Abp.Domain.Entities.Auditing;
using AppointmentApp.Slots;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppointmentApp.Advisers
{
    public class Adviser : FullAuditedEntity<int>
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string CompanyName { get; set; }

        public virtual ICollection<Slot> Slots { get; set; }
    }
}

File: appointments-mobile/src/shared/models/Adviser.ts
import { Slot } from "./Slot";

export class Adviser {
    public id: number;
    public firstName: string;
    public lastName: string;
    public phoneNumber: string;
    public companyName: string;
    public fullName: string;

    public Slots: Array<Slot>;
    public slotMaps: Map<string, Slot>;

    constructor(){
        this.Slots = new Array<Slot>();
        this.slotMaps = new Map<string, Slot>();
    }
}